src/lib/rule-finder.js

```diff
@@ -78,10 +78,9 @@ function _isNotCore(rule) {
   return rule.indexOf('/') !== '-1';
 }
 
-function RuleFinder(specifiedFile, options) {
-  const {omitCore, includeDeprecated} = options;
+function RuleFinder(specifiedFile, {omitCore, includeDeprecated, ext = ['.js']}) {
   const configFile = _getConfigFile(specifiedFile);
-  const files = glob.sync('**/*.js', {dot: true, matchBase: true});
+  const files = glob.sync(`**/*@(${ ext.join('|') })`, {dot: true, matchBase: true});
```

Comment:
can we ensure that the extensions don't contain `*`, or `./` or `../`? also, does eslint allow an ext like `.a.b`, or must it be a single one?

Reply:
I will do a post filter instead then. I think that eslint allow multi dots extensions but I am not 100% sure.